pages/applications.tsx

```diff
@@ -0,0 +1,192 @@
+// Copyright (C) 2023 Nethesis S.r.l.
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+import type { NextPage } from 'next'
+import { useTranslation } from 'react-i18next'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {
+  faArrowRight,
+  faPhoneVolume,
+  faCameraSecurity,
+  faVideo,
+  faChartSimple,
+  faArrowUpRightFromSquare,
+} from '@nethesis/nethesis-solid-svg-icons'
+import { Button } from '../components/common'
+import { faPhone } from '@fortawesome/free-solid-svg-icons'
+import { getApiEndpoint } from '../lib/utils'
+import { getApiScheme } from '../lib/utils'
+
+const Applications: NextPage = () => {
+  const { t } = useTranslation()
+  const pathCompany = getApiEndpoint()
+  const apiScheme = getApiScheme()
+  const linkNethvoiceReport = apiScheme + pathCompany + '/pbx-report/'
```

Comment:
Let's not use "Nethvoice" and other product names in the code. I would rename this variable to `pbxReportUrl`